DataDog/apps
# chore: remove VS Code Workspace Settings

## Motivation

- Return Workspace-level Setting control to users

## Changes

- Remove extraneous `.vscode/settings.json` file and `.gitignore` it so developers have control over their own VS Code Workspace Settings

## Testing

- Open a TypeScript file in VS Code
- Make some changes to it
- Save and (if you have Format on Save enabled in your Settings) notice it formats properly based on our Prettier settings

## Releases

Choose one:

-   [x] No release is necessary.
        If you're only updating examples/documentation, this is likely what you want.
-   [ ] All packages that need a release have a changeset.
